osbs/build/build_response.py

```diff
@@ -97,7 +97,7 @@ def get_logs(self, decode_logs=True):
         """
         logs = graceful_chain_get(self.get_annotations_or_labels(), "logs")
         if not logs:
-            logger.error("no logs")
+            logger.warn("no logs found in annotations")
```

Comment:
If we expect this to normally happen with future builds then let's use some other log level - debug?